src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -247,11 +247,16 @@ export class EditConcept extends Component {
   updateEventListener = (event, url) => {
     const { value, name } = event.target;
     const { mappings } = this.state;
+    const eventName = name === 'source';
```

Comment:
In how many places are using "source"?

Reply:
I am using it in one place.

Reply:
Are you sure?

Reply:
Yes I am.

Reply:
That is not true

Reply:
I have double checked and I cannot find any other occurrence of that expression.

Reply:
Can you use a constant for it?

Reply:
I have changed it to a constant